mjpc/CMakeLists.txt

```diff
@@ -12,6 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+include(FetchContent)
+FetchContent_Declare(json URL https://github.com/nlohmann/json/releases/download/v3.11.3/json.tar.xz)
```

Comment:
@nimrod-gileadi is there a preferred Google solution for the json functionality needed for this PR?

Reply:
I realized for the current scope of this PR I don't need to include this library. For the future PRs it would be good to know if you recommend any alternative JSON parser libraries.